### ISSUE TITLE
#将ghostnet训练place365分类效果不够理想的问题#

### ISSUE THREAD

**Issue by Geroge-W** [huawei-noah/ghostnet#44]
@iamhankai 您好，我尝试了将ghostnet网络在place365数据集上进行微调，在迭代了100个epoch之后，Acc@1: 35.% ,Acc@5: 62.%。
部分超参数设置如下：
batch_size = 256;
loss: CrossEntropyLoss();
optimizer: SGD();
lr_init=0.1 & Sets the learning rate to the initial LR decayed by 10 every 30 epochs

先只对最后的classifier训练，后面又对整个网络进行了训练，得到的两者的结果都很相近。
请问您能给一些在place365数据集上训练时的建议和帮助吗？
非常感谢！
